_data/simple-icons.json

```diff
@@ -250,6 +250,11 @@
             "hex": "008373",
             "source": "https://commons.wikimedia.org/wiki/File:Bing_logo_(2016).svg"
         },
+        {
+            "title": "Bit",
+            "hex": "703A8E",
```

Comment:
It seems that purple is indeed their brand color :stuck_out_tongue_winking_eye: May I ask where you got this particular color from?

Reply:
I got it from the logo:
https://storage.googleapis.com/bit-docs/bit-logo%402x.png

Reply:
Hmm, looks like you're in the [bit.dev organisation](https://github.com/teambit) on GitHub. Any chances you can get a better source for the color? If not that's fine, after all you're here to represent the brand I guess 😉 

The reason I'm asking is that extracting a color from a PNG (or JPG, or whatever) is not very reliable due to compression. If you have a "more specific" color, for example from your CSS or from some internal document, that would be beneficial to you.

Reply:
sure, 
I take it now from our color palette  :)

Reply:
Super 👍